package.json

```diff
@@ -8,7 +8,8 @@
 		"build": "next build",
 		"start": "next start",
 		"lint": "next lint",
-		"format": "prettier --write ."
+		"format": "prettier --write .",
```

Comment:
I prefer to use the prettier ESLint plugin for formatting so that way `eslint --fix` applies the linting fixes in addition to the formatting errors all in one go. A dev can use an eslint IDE integration to get auto formatting and linting fixes on file save.